pkg/api/handlers/tests/feature_flag_handler_test.go

```diff
@@ -48,6 +49,10 @@ func (suite *FeatureFlagHandlerTestSuite) SetupTest() {
 		middlewares.AuthMiddleware(h.PatchFeatureFlag),
 	)
 	suite.Server.GET("/organization/:organizationID/feature-flag", middlewares.AuthMiddleware(h.ListFeatureFlags))
+	suite.Server.PATCH(
+		"organization/:organizationID/feature-flag/:featureFlagID/revision/:revisionID",
```

Comment:
```suggestion
		"/organization/:organizationID/feature-flag/:featureFlagID/revision/:revisionID",
```

nitpick fix through github